addons/html_editor/static/tests/_helpers/editor.js

```diff
@@ -160,7 +165,14 @@ export async function testEditor(config) {
         };
     }
     delete config.props?.mobile;
-    const { el, editor } = await setupEditor(contentBefore, config);
+    let resolveWillBeDestroyed;
+    const willBeDestroyed = new Promise((resolve) => {
```

Comment:
Prefer the use of Deferred from `@web/core/utils/concurrency`.
This class already implement a callable resolve directly on the object.